src/Avalonia.Diagnostics/Diagnostics/Views/MainWindow.xaml

```diff
@@ -12,6 +12,9 @@
     <StyleInclude Source="resm:Avalonia.Themes.Default.DefaultTheme.xaml?assembly=Avalonia.Themes.Default"/>
     <StyleInclude Source="resm:Avalonia.Themes.Default.Accents.BaseLight.xaml?assembly=Avalonia.Themes.Default"/>
     <StyleInclude Source="resm:Avalonia.Controls.DataGrid.Themes.Default.xaml?assembly=Avalonia.Controls.DataGrid"/>
+    <Style Selector="ContextMenu">
```

Comment:
Can you use more specific selector?
Maybe something like "DataGrid ContextMenu"